packages/core/src/asset/mintDrop.ts

```diff
@@ -0,0 +1,62 @@
+import type { Signer } from 'ethers'
+import invariant from 'ts-invariant'
+import { checkOwnership, pollOwnership } from '../exchange/offerQuantityChanges'
+import type { Sdk } from '../graphql'
+import type { IState, TransactionHash, UUID, Uint256 } from '../types'
+import { toAddress, toTransactionHash } from '../utils/convert'
+import { sendTransaction } from '../utils/transaction'
+
+export type State =
+  | IState<'TRANSACTION_SIGNATURE', {}>
+  | IState<'TRANSACTION_PENDING', { txHash: TransactionHash }>
+  | IState<'OWNERSHIP', {}>
+
+export async function mintDrop(
+  sdk: Sdk,
+  dropId: UUID,
+  quantity: Uint256,
+  signer: Signer,
+  onProgress?: (state: State) => void,
+): Promise<UUID> {
```

Comment:
I think this function should return the IDs of the assets minted and not the drop.